### PR TITLE
input-autocomplete-onoff: add search keywords

### PR DESCRIPTION
**Pull request by cvrebert**

### CODE REVIEW

**Comment by Fyrd** [comment]
Actually the text inside the title and description are already used for searches, so need to duplicate them like this.

**Comment by cvrebert** [comment]
@Fyrd Ah, gotcha. Is there a reason the entry is still `"shown":false` ?

**Comment by Fyrd** [comment]
Yeah... :-/ Turns out it's a rather complicated and controversial attribute and from the research I started doing it's hard to tell whether latest changes by browsers count as them following the spec or not.

Most times it comes down to devs wanting browsers to honor "off" in all cases with browser vendors wanting to override it for the purpose of autofilling, etc. to make users happier and for security reasons. I've seen the argument made that they can do that and still follow the spec, but not 100% sure it's true. 

So basically just need to do more research before I'm willing to commit to the current data as it stands.

**Comment by cvrebert** [comment]
Understood. Thanks for all the work you do!
